elementary/exceptions/exceptions.py

```diff
@@ -82,6 +82,17 @@ def extract_detailed_dbt_command_args(command_args):
         except Exception as ex:
             logger.error(f"Failed to extract detailed dbt command args, error: {ex}")
 
+    def get_exception_message(self) -> Optional[str]:
+        error_output = self.proc_err.output.decode()
```

Comment:
`proc_err.output` can potentially be `None` if `check_output=False`, maybe worth adding a check here.
I think that we should raise an error in this case rather than return `None` as this is a coder problem.